packages/server/src/api/bannerRouter.ts

```diff
@@ -98,20 +98,32 @@ export const buildBannerRouter = (
                 targetingMvtId,
             );
 
+            // const separateArticleCountSettings = variant.separateArticleCountSettings ?? {
+            //     type: 'above',
+            //     copy: 'You’ve read %%ARTICLE_COUNT%% articles in the last month.',
+            // };
+
+            const countType = !variant.separateArticleCountSettings?.copy
```

Comment:
can this be removed now? I don't think we're using `countType` client-side

Reply:
Removed countType